### PR TITLE
you can now play cats without being catbanned

### PR DESCRIPTION
**Pull request by ghost**
\*snuggles u\*
uguuu~

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
**MAIM KILL BURN!! MAIM KILL BURN!!**

**Comment by PopNotes** [comment]
It's a yes from me. I don't want to have to break rules in order to unlock hard mode.

**Comment by PopNotes** [comment]
Give me, give me, chicken tendies, be they crispy or from Wendy's.
Spend my hard-earned good boy points on Kid's Meal ball pit burger joints.
Mommy lifts me to the car to find me tendies near and far.
Enjoy my tasty tendie treats in comfy big boy booster seats.
McDonald's, Hardee's, Popeye's, Cane's- but of my tendies none remains.
She tries to make me take a nappy but sleeping doesn't make me happy.
Tendies are the only food that puts me in the napping mood.
I'll scream, I'll shout, I'll make a fuss! I'll scratch, I'll bite, I'll even cuss!
Tendies are my heart's desire, fueled by raging, hungry fire.
Mommy sobs, and wails, and cries but tears aren't tendies, nugs or fries.
My good boy points were fairly earned to buy the tendies that I've yearned.
But there's no tendies on my plate, did mommy think that I'd just ate?!
Tendies, tendies, get them now you fat, ungrateful, sluggish sow! REEEEEEEEEEEEEEE!
I screech while hurling into her eyes my foul-smell bowel-dwelling diaper surprise!
For she who is unpooped on is she who remembers: Never forget my chicken tenders.

**Comment by ghost** [comment]
I thought people hated cats rolling antag though.
I don't see why people should play a race that disallows antags rolls while playing.

**Review by KayeArray — APPROVED** [review]
code is fine

**Comment by PopNotes** [comment]
The only people that hated cats rolling antag were the ones who underestimated catbeasts and got pooped on for it.

**Comment by GuyonBroadway** [comment]
Nah it was cus people would use it as a free way to die each round if they didn't roll antag and there is no real way to police that shir.

**Comment by PopNotes** [comment]
The answer is simple, don't let catbeasts roll finite jobs.

**Comment by ghost** [comment]
that's not what guyon is trying to say
he's trying to say catbeasts were used to roll for antag since you could die ICily every time and it wasn't easily punishable

**Comment by KayeArray** [comment]
It was abused to hell and back by Roma basically.

**Comment by PopNotes** [comment]
If one specific player is causing so much trouble by abusing the feature when practically nobody else of note has done it to such a degree, should we really change the fucking code itself to suit them instead of just unzipping rule 0 and removing the problem from its roots?

**Comment by Pyko1** [comment]
sounds like a bug to me if you could have catshit selected and roll for antag.
Catshits should be limited to assistants, and have all antag preferences forced to no.

**Comment by tortellinitony** [comment]
Then we'd have to add more checks for rolling antag, no thanks. I prefer that races be cosmetic only, anyway

**Comment by ghost** [comment]
I don't like denying players antagonist for picking a race, especially if people aren't aware of playing "hardmode" prevents you from rolling.
I also believe a lot players dislike antagonist cats.

**Comment by ghost** [comment]
> I also believe a lot players dislike antagonist cats.

Again, that's because they believe "hehehe ez valids" and then get robusted hard

**Comment by PopNotes** [comment]
>I also believe a lot players dislike antagonist cats.

I think that's just too bad for those players, then.
This is, after all, a game about being worried as to who might nail you in the face with an ebow when you turn a corner. Adding that extra layer of paranoia as to whether or not you should chase that catshit into maint only improves things in my eyes.

**Comment by GuyonBroadway** [comment]
The two big points where number 1.

Its a punishment, letting players opt to start with a punishment dilutes the nature of the punishment. Something I saw was people going out their way to rescue cats because of a horrible anti-meme that had built up over time. Catbeasts ended up getting anti-lynch squads who would save their butts and let them loose in some hidden corner of maint because its just as much a challenge to save a valid without breaking the rules as it is to not die as a valid. 

Second is the problem of people using it to dodge the "no suicide over and over if you don't get antag" rule. Sure it was Roma who did it a lot but he highlighted the problem in broader scope in that there isn't really any way to differentiate between a cat player being killed and a cat player letting themselves be killed because they want to die short of observing the fuckers round in round out and quizzing them every time they die in a way that doesn't look like they genuinly tried to avoid being murdered. 

 tl:dr its just an administrative clusterfuck.

**Comment by Supermichael777** [comment]
or you could make cats not infinitely valid./s

**Comment by GuyonBroadway** [comment]
@Supermichael777
Short answer no

Long aswer noooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

**Comment by Swagile** [comment]
yeah, we still have this problem even after round start cat was removed

we have people specifically who go out of their way to help cats, even stunning crew who attack cats, and dragging them to safety

thats fine and all, since its the nature of the game, but i don't want it to become a daily thing (and it WILL become daily if roundstart cat is made a thing)

plus the second issue guyon highlighted is hugely why this is a BAD IDEA

**Comment by ghost** [comment]
this just removes the culture of hippie and makes us even less unique. Plus cats are cancer.  You should make cats locked to assistants and never antagonists instead,  making it even more of a punishment edit: i mean if cats aren't valid.  Also let us play cats anytime just make them always nonantag and locked to assistant

**Comment by PopNotes** [comment]
The thing is, catbanned players are ALSO antagbanned by extension. Simply playing as a cat still lets you roll antag, so it's not like catbanning loses its punishment status just by also allowing people to select tajaran as their race.

The only real solution I can see to the second problem that doesn't just straight up disable catting for normal players is to limit cat roles to assistant or otherwise unimportant roles like janitor or clown.

**Comment by PopNotes** [comment]
>this just removes the culture of hippie and makes us even less unique. Plus cats are cancer. You should make cats locked to assistants and never antagonists instead, making it even more of a punishment

This is doubleplus wrong, as this doesn't make cats non-valid. THAT would remove some of hippie culture, the well-known kitty lynch squads. The problem is that we almost NEVER see cats anymore, since the actual rules behind catbanning are very teetery and most people that get catbanned just stop playing the game until their timer runs out. It's not like you'd see more than 2-3 cats per round back when this was a feature anyway, since most people DON'T want to be permanently valid.

Re-adding cats at roundstart will re-invigorate that lost artform of literal cat-and-mouse-but-the-mouse-is-the-cat chases.

**Comment by ghost** [comment]
I agree roundstart being able to be catbeast would be good BUT make cats never antags,  and only assistant,  maaaaybe service jobs,  to minimize roma RTA catbeast shittery.  Or just warn roma he'll be banned if he does that shit. 
Roundstart catbeast is good i aggree

**Comment by PopNotes** [comment]
I disagree with the 'no antag for non-banned cat players' method, because playing antag cat was genuinely some of the most fun I've ever had with this game.

**Comment by ghost** [comment]
Just have it not be able to be abused by roma to roll antag over and over

**Comment by PopNotes** [comment]
Again, the answer is simple, rule 0 Roma.

**Comment by Pyko1** [comment]
make the catshit antags only know they are antags after 5 mins or dying.

**Comment by GuyonBroadway** [comment]
Banning Roma does not fix the problem.

**Comment by Pyko1** [comment]
so true the issue can be fixed with code if we can just agree on it.

**Comment by PopNotes** [comment]
>Banning Roma does not fix the problem.

It fixes a few problems and remedies this one greatly, since he's the only person I've ever seen doing it.

**Comment by Pyko1** [comment]
punishing people for bad code is bad tho

**Comment by PopNotes** [comment]
it has nothing to do with code tho

**Comment by Swagile** [comment]
roma is the only person known doing it A LOT, and he's a high profile player so his actions are noticed much more widely than if, say, random joe blow69 joins the server and does the same thing.

i know a few people who have done this type of thing on and off, and its very hard to prove from an administrative stand point unless its a full blown trend that other players notice and complain about, aka roma

**Comment by GuyonBroadway** [comment]
The thing is if roma can do it, anyone can do it and people will do it. The only reason we know Roma does it is because, as Swagile said, Roma is high profile and his actions are scrutinized more. Other players will do it too and it will be even harder to spot.

**Comment by j-mie** [comment]
This doesn't work on Hippie, never has IMO. Antag cats are way too bad.
